packages/api/client.ts

```diff
@@ -156,16 +158,34 @@ export const getStops = async (dbClient: Kysely<Database>, input: StopsQueryInpu
                     ),
                 ),
         )
-        .where("stopType", "not in", ignoredStopTypes)
-        .where("status", "=", "active")
+        .$if(!!input.stopTypes?.[0], (qb) => qb.where("stopType", "in", input.stopTypes ?? ["---"]))
+        .$if(!!input.busStopType, (qb) => qb.where("busStopType", "=", input.busStopType ?? "---"))
         .offset((input.page || 0) * STOPS_PAGE_SIZE)
         .limit(STOPS_PAGE_SIZE)
         .execute();
 
     return stops;
 };
 
-export type Stops = Awaited<ReturnType<typeof getStops>>;
+export type Stops = {
```

Comment:
Can we add a comment as to why Awaited line was removed?

Reply:
done